package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurant-js",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --progress",
+    "start": "webpack-cli serve --mode development",
+    "watch": "webpack --progress --watch --mode=development"
+  },
```

Comment:
I was unable to run any of your script commands. Please test them to ensure that they execute without errors. Here is a basic example configuration ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```